workspaces/cli-server/src/spectacle/index.ts

```diff
@@ -50,8 +51,22 @@ export class LocalCliSpecRepository implements IOpticSpecReadWriteRepository {
 
   constructor(private dependencies: LocalCliSpecRepositoryDependencies) {
     this.notifications = dependencies.notifications;
+    this.initialize();
   }
 
+  initialize() {
+    fs.watch(
+      this.dependencies.specStorePath,
```

Comment:
`fs.watch` (proxied by `fs-extra` directly from `fs`) [is pretty unreliable](https://github.com/paulmillr/chokidar#why), especially on macOS as ti doesn't use the OS's native FSEvents API :

> Node.js fs.watch:
> 
> - Doesn't report filenames on MacOS.
> - Doesn't report events at all when using editors like Sublime on MacOS.
> - Often reports events twice.
> - Emits most changes as rename.
> - Does not provide an easy way to recursively watch file trees.
> - Does not support recursive watching on Linux.
> 
> Node.js fs.watchFile:
> 
> - Almost as bad at event handling.
> - Also does not provide any recursive watching.
> - Results in high CPU utilization.

Fortunately, there's modules like [chokidar](https://www.npmjs.com/package/chokidar) or [fsevents](https://www.npmjs.com/package/fsevents) that do this properly.